rest-api/tools/import_organizations.py

```diff
@@ -142,7 +142,7 @@ def _cleanup_old_entities(self, session, row_list):
           hpo_id_list.append(old_hpo.hpoId)
           self.deletion_count += 1
 
-      if hpo_id_list:
+      if hpo_id_list and not dry_run:
```

Comment:
Please add (dry run) to log statements in these methods

Reply:
done